src/modules/polkadotSlotAuction/components/SupportProject/components/SupportProjectForm/SupportProjectForm.tsx

```diff
@@ -14,23 +14,39 @@ import { Body1 } from '../../../../../../UiKit/Typography';
 import { Button } from '../../../../../../UiKit/Button';
 import { FormErrors } from '../../../../../../common/types/FormErrors';
 import { useSlotAuctionSdk } from '../../../../hooks/useSlotAuctionSdk';
+import { INDEX_PATH } from '../../../../../../common/const';
+import { QueryLoading } from '../../../../../../components/QueryLoading/QueryLoading';
+import { useCrowdloansWithBalances } from '../../../../hooks/useCrowdloans';
+import { getBalance } from '../../../../utils/getBalance';
 
 export const SupportProjectForm = () => {
   const classes = useSupportProjectFormStyles();
-  const balance = 47;
+  const [isLoading, setIsLoading] = useState(false);
   const dailyReward = 1.25;
   const initialReward = 825.25;
 
   const { id } = useParams<{ id: string; name: string }>();
   const { slotAuctionSdk, polkadotAccount } = useSlotAuctionSdk();
+  const { balances } = useCrowdloansWithBalances(
+    slotAuctionSdk,
+    'SUCCEEDED',
+    polkadotAccount,
+  );
+  const balance = getBalance(balances, Number.parseInt(id));
 
   const handleSubmit = async (payload: FormPayload) => {
-    console.log(payload);
-    await slotAuctionSdk.depositFundsToCrowdloan(
-      polkadotAccount,
-      Number.parseInt(id),
-      new BigNumber(`${payload.contributeValue}`),
-    );
+    setIsLoading(true);
+    try {
+      await slotAuctionSdk.depositFundsToCrowdloan(
+        polkadotAccount,
+        Number.parseInt(id),
+        new BigNumber(`${payload.contributeValue}`),
+      );
+    } catch (e) {
+    } finally {
+      setIsLoading(false);
+      window.location.replace(INDEX_PATH);
```

Comment:
u should't reload here

```suggestion
      window.history.replaceState(null, '', INDEX_PATH);
```